repos/fdbt-site/tests/pages/api/fareType.test.ts

```diff
@@ -1,4 +1,4 @@
-import { CAPPED_PRODUCT_ATTRIBUTE, TXC_SOURCE_ATTRIBUTE } from './../../../src/constants/attributes';
+import { TXC_SOURCE_ATTRIBUTE } from './../../../src/constants/attributes';
 import { getMockRequestAndResponse } from '../../testData/mockData';
 import fareType from '../../../src/pages/api/fareType';
```

Comment:
Doesnt look like we have any tests to show that it now redirects the user differently?